migrations/0025-add-public-activity-scope-index.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner, TableIndex } from 'typeorm';
+
+export class AddActivityScopeIndex1608574428247 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createIndex('Activities',
+      new TableIndex({
+        name: 'public_activities_by_user_index',
+        columnNames: ['scope'],
+        where: 'scope = \'PUBLIC\'',
+      }));
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.dropIndex('Activities', 'public_activities_by_user_index');
```

Comment:
This should restore the old index.

Reply:
This migration didn't actually drop the old index (the previous one did), so if we restore it here then if we rollback just this migration and not the last, there would be an error since the `public` column referenced by that old index doesn't exist